version.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
-    <VersionPrefix>2.1.0</VersionPrefix>
-    <VersionSuffix>rc1</VersionSuffix>
+    <VersionPrefix>2.2.0</VersionPrefix>
```

Comment:
Why is this `2.2` ?

Reply:
The PR was originally based off of dev before it was changed to 2.2.0. About to send an update to base it off of release/2.1 branch.